udev/CMakeLists.txt

```
@@ -1,3 +1,8 @@
 install(FILES 50-revpi.rules
   DESTINATION /lib/udev/rules.d
 )
+
+install(FILES revpi_mac
+  DESTINATION /usr/lib/udev
+  PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
```

```suggestion
install(PROGRAMS revpi_mac
  DESTINATION ${CMAKE_INSTALL_LIBDIR}/udev
```
Using PROGRAMS should solve the permission issues: https://cmake.org/cmake/help/latest/command/install.html#installing-files

Useing the ${CMAKE_INSTALL_LIBDIR} from the GNUInstallDirs would be advisable. Since Buster /lib is a symlink to /usr/lib so this works fine either way. And the usual /usr prefix can be set by the debianization or the user.

---

I changed that. BUT this might break the udev rule if the path is not given correctly, because udev only has /usr/lib/udev in PATH, not /usr/local/lib/udev.

---

The manual says `/usr/local/lib/udev/rules.d` is okay. So I would think `/usr/local/lib/udev/` should be okay as well.
https://www.freedesktop.org/software/systemd/man/udev.html#Rules%20Files